README.md

```diff
@@ -1,25 +1,55 @@
-# Axiom CloudWatch Lambda [![CI](https://github.com/axiomhq/axiom-cloudwatch-lambda/actions/workflows/ci.yaml/badge.svg)](https://github.com/axiomhq/axiom-cloudwatch-lambda/actions/workflows/ci.yaml)
+# Axiom CloudWatch Forwarder [![CI](https://github.com/axiomhq/axiom-cloudwatch-forwarder/actions/workflows/ci.yaml/badge.svg)](https://github.com/axiomhq/axiom-cloudwatch-forwarder/actions/workflows/ci.yaml)
 
-Axiom CloudWatch Lambda is an easy-to-use AWS CloudFormation template to send logs from CloudWatch to [Axiom](https://axiom.co). It deploys a CloudWatch log group subscription filter and a Lambda.
+Axiom CloudWatch Lambda is an easy-to-use AWS CloudFormation template to send logs from CloudWatch to [Axiom](https://axiom.co). It deploys a Lambda and a subscriber to create the needed CloudWatch log group subscription filters.
 
 Axiom CloudWatch Lambda uses the following CloudFormation stacks:
 
-- Axiom Ingester creates a Lambda function that ingests logs from CloudWatch and sends them to Axiom.
-- Backfiller runs once to create subscription filters on the ingest Lambda for all existing CloudWatch log groups.
-- Logs Subscriber creates a Lambda function that listens for new log groups and creates subscription filters for them. This way, you don't have to create subscription filters manually for new log groups.
+- Axiom Forwarder creates a Lambda function that ingests logs from CloudWatch and sends them to Axiom.
+- Subscriber runs once to create subscription filters on the ingest Lambda for all existing CloudWatch log groups.
+- LogGroups Listener creates a Lambda function that listens for new log groups and creates subscription filters for them. This way, you don't have to create subscription filters manually for new log groups.
```

Comment:
```suggestion
- _Listener_ creates a Lambda function that listens for new CloudWatch log groups specified by name, pattern, and/or regular expression. It will create subscription filters for them on the _Forwarder_ Lambda. This way, you don't have to create subscription filters manually for new log groups.
```